specs/plugin-extensibility.md

```diff
@@ -146,9 +154,18 @@ All response attributes are required.
   // List of contract versions supported by the plugin, one per major version
   "supportedContractVersions" : [ ],
   
-  // Currently one of 
-  // SIGNATURE_GENERATOR or
-  // SIGNATURE_ENVELOPE_GENERATOR
+  // List of one or more capabilities supported by plugin.
+  // Valid values are 
+  //    SIGNATURE_GENERATOR.RAW
+  //    SIGNATURE_GENERATOR.ENVELOPE
+  //    SIGNATURE_VERIFIER.TRUSTED_IDENTITY
+  //    SIGNATURE_VERIFIER.REVOCATION_CHECK
```

Comment:
Will we support more `SIGNATURE_VERIFIER` capabilities? like `SIGNATURE_VERIFIER.RAW` and `SIGNATURE_VERIFIER.ENVELOPE`?

Reply:
The intent is to only allow specific verification steps in the overall workflow to be offloaded to plugin.  We can extend this interface to offload step 4 onwards in [verification workflow](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#signature-verification) as required. We can also add steps like Trust Store resolver, Key resolver etc. as needed. 
I'm not in favor of adding a SIGNATURE_VERIFIER.ENVELOPE which will offload full verification unless we have a justified use case, we want notation to perform basic validation wherever possible.